src/client/containers/Login/Login.js

```diff
@@ -0,0 +1,79 @@
+import React , {Component} from 'react';
+import { Link } from 'react-router-dom';
+import CssBaseline from '@material-ui/core/CssBaseline';
+import Avatar from '@material-ui/core/Avatar';
+import Button from '@material-ui/core/Button';
+import TextField from '@material-ui/core/TextField';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import Checkbox from '@material-ui/core/Checkbox';
+import Grid from '@material-ui/core/Grid';
+import Paper from '@material-ui/core/Paper';
+import Typography from '@material-ui/core/Typography';
+import classes from './Login.css';
+
+export default class Login extends Component {
```

Comment:
If you are creating component without state its better to create not a class component, here i'd suggest to use just function